Broken semver

A patch version of redbox-react was published an hour ago but the peer dependency was changed to only support React 15.

Our CI is now failing with:

```
error	17-Jun-2016 16:17:59	npm ERR! peerinvalid The package react@0.14.8 does not satisfy its siblings' peerDependencies requirements!
...
error	17-Jun-2016 16:17:59	npm ERR! peerinvalid Peer redbox-react@1.2.7 wants react@^15.0.0
...
```

Can we change this to a major version bump? Dropping support for React 0.14 is a breaking change.
It would be good if we could get a patch version on the v1 branch which is just an old version which supports React 0.14 so that we don't break compatibility for other users.

Thanks

Same issue here. After uploading a new container, this issue occured during building:

´´´
npm ERR! peer dep missing: react@^15.0.0, required by redbox-react@1.2.7
npm ERR! peer dep missing: react@^15.1.0, required by react-dom@15.1.0
´´´

And then react-transform-catch-errors would stop working. Just fix the version to the previous (or upgrade react to avoid this issues.

A project I'm working on is using React 0.14 and has recently started failing on CI with the error message:

```
Error: The package react does not satisfy its siblings' peerDependencies requirements!
```

I ran `npm ls react-dom` from the root project directory:

```
├─┬ babel-preset-react-hmre@1.1.1
│ └─┬ redbox-react@1.2.8
│   ├── UNMET PEER DEPENDENCY react@^15.1.0
│   └── react-dom@15.1.0
└── react-dom@0.14.8

npm ERR! peer dep missing: react@^15.1.0, required by react-dom@15.1.0
```

It looks like v1.2.8 of redbox-react requires [`react-dom: ^15.1.0` as a dependency](https://github.com/KeywordBrain/redbox-react/blob/v1.2.8/package.json#L59), and I think this may be the problem.

I cloned `redbox-react` and `babel-preset-react-hmre` and made the following changes locally:

- `redbox-react`: loosened the `react-dom` dependency: `"react-dom": "^0.14.0 || ^15.0.0"`
- `babel-preset-react-hmre`: changed `package.json` to use the updated `redbox-react` dependency
- my project: changed `package.json` to use the updated `babel-preset-react-hmre` dependency

I ran `rm -rf node_modules && npm install` on my project locally, and it succeeded. I'll open a PR with the changes. 🎉

--- a/package.json
+++ b/package.json
@@ -56,6 +56,6 @@
   "dependencies": {
     "error-stack-parser": "^1.3.6",
     "object-assign": "^4.0.1",
-    "react-dom": "^15.1.0"
+    "react-dom": "^0.14.0 || ^15.0.0"
   }
 }

